### ISSUE TITLE
PLoS template mfailing with \citep

### ISSUE THREAD

**Issue by sguyader** [rstudio/rticles#55]
I tried to knit the PLoS template in RStudio, but it failed with the following error in the R Markdown output console:
```
output file: Untitled.knit.md
! Undefined control sequence.
l.114 Cite fancy references \citep     
                                       {Garnier2007}.
```
I think it is because there should be \usepackage{natbib} in the template.tex, but there is none.
When adding this line to template.tex, it compiles and produces the expected result.

**Comment by jjallaire** [comment]
@yihui Is this in some way related to the recent LaTeX bibliography work you have done?

**Comment by yihui** [comment]
No, I don't think so. I think it is because the PLoS template uses the **cite** package: https://github.com/rstudio/rticles/blob/65517dd2dc2fe0059bcc46fae5588c5ecd31d466/inst/rmarkdown/templates/plos_article/resources/template.tex#L29-L30, which does not understand `\citep`. Pandoc is more friendly to **natbib** than **cite**. I guess you have to use either **natbib** or plain `\cite` (generated from `@Garnier2007` instead of `[@Garnier2007]`).

**Comment by yihui** [comment]
Sorry it was indeed my problem. I have fixed it via rstudio/rmarkdown#578.
